test/spec/cloud-element-templates/fixtures/condition.json

```diff
@@ -72,7 +72,8 @@
       "type": "String",
       "value": "nameProp=foobar",
       "binding": {
-        "type": "zeebe:taskDefinition:type"
+        "type": "zeebe:taskDefinition",
```

Comment:
We remove the legacy test here, don't we? 🤔

Reply:
This is to check interoperability of legacy and new way, cf. https://github.com/bpmn-io/bpmn-js-element-templates/pull/29/files/d7feba35e087b75c5386fdd457d2059ae3e665dd#diff-53a990bb6b5fec34594754a48dc757a1ee38f419931c0a82b379d8fd4b4cb2a1R63
I could add another property instead, if it makes sense.

Reply:
Okay, understood. 👍🏻